### PR TITLE
INC-1182 Removing old incentives behaviour entries table

### PR DESCRIPTION
**Pull request by ashleygyngell**

### CODE REVIEW

**Comment by ushkarev** [comment]
There are a fair few other things can be removed as well. Let's chat later

File: server/routes/incentivesTable.ts [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-    const agencyId = locationPrefix.split('-')[0]
-
-    const systemToken = await hmppsAuthClient.getSystemClientToken(user.username)
-
-    const behaviorService = new BehaviourService(systemToken)
-    const entries = await behaviorService.getLocationSummary(agencyId, locationPrefix)
-
-    const threeMonthsAgo = daysAgo(90)
-
-    res.render('pages/incentives-table', { entries, threeMonthsAgo, locationPrefix, feedbackUrl })
```
Comment (neilmendum) [inline server/routes/incentivesTable.ts:30]: I think you'll also want to delete `incentives-table.njk`

File: server/config.ts [hunk 2]
```diff
@@ -135,7 +135,6 @@ export default {
     addTestErrorEndpoint: flag('FEATURE_ADD_TEST_ERROR_ENDPOINT', false),
     // Whether to hide the 'Days since last review'/'Days on level' columns
     hideDaysColumnsInIncentivesTable: flag('FEATURE_HIDE_DAYS_COLUMNS_IN_INCENTIVES_TABLE', false),
```
Comment (neilmendum) [inline server/config.ts:137]: I think we can also remove this feature flag, as it was only used by `incentivesTable`

**Review by neilmendum — COMMENTED** [review]
Looks good! Added two comments

File: server/config.ts [hunk 3]
```diff
@@ -133,9 +133,6 @@ export default {
   },
   featureFlags: {
     addTestErrorEndpoint: flag('FEATURE_ADD_TEST_ERROR_ENDPOINT', false),
-    // Whether to hide the 'Days since last review'/'Days on level' columns
-    hideDaysColumnsInIncentivesTable: flag('FEATURE_HIDE_DAYS_COLUMNS_IN_INCENTIVES_TABLE', false),
```
Comment (ushkarev) [inline server/config.ts:137]: `FEATURE_HIDE_DAYS_COLUMNS_IN_INCENTIVES_TABLE` was only used in the old table, so i think it's correct to remove it.

it still appears in the various helm_deploy/values files and some .env files too so i'd get delete it there too

**Review by xoen — APPROVED** [review]
Nice one @ashleygyngell - love those cleanup PRs 😍

**Pull request merged**